src/dlgpreflibrary.cpp

```diff
@@ -59,7 +59,7 @@ DlgPrefLibrary::DlgPrefLibrary(QWidget * parent,
 
     // plugins are loaded in src/main.cpp way early in boot so this is safe
     // here, doesn't need done at every slotUpdate
-    QStringList plugins(SoundSourceProxy::supportedFileExtensionsByPlugins());
+    QStringList plugins(SoundSourceProxy::getSupportedFileTypesByPlugins());
```

Comment:
There already was an inconsistency between "type" and "extension" before I started to refactor the SoundSource API. I have chosen the more generic term "type" instead of "extension", because in the future we might improve the deduction of the file type. For the time being simply the file extension is used as the type of the file.